sdk/appconfiguration/app-configuration/samples/react-sample/sample.env

```diff
@@ -1 +1,14 @@
-REACT_APP_APPCONFIG_CONNECTION_STRING=<connection-string>
+# Rename this to .env in order to have React automatically
```

Comment:
Just checking - is it React that's loading this .env file? Or is it _us_ loading it and then exposing it to React?

Reply:
It is react, not us.